seqmagick/subcommands/quality_filter.py

```diff
@@ -11,16 +11,29 @@
 import time
 
 from Bio import SeqIO
-try:
-    from Bio import trie, triefind
-except ImportError:
-    trie = None
-    triefind = None
+import pygtrie as trie
 from Bio.SeqIO import QualityIO
 
 from seqmagick import fileformat, __version__
 from .common import typed_range, FileType
 
+
+def trie_match(string, trie):
```

Comment:
From my limited knowledge about this code base, this function is a perfect replacement for `triefind.match`. Good work!